tests/select_related/tests.py

```diff
@@ -163,6 +163,18 @@ def test_chaining(self):
             self.assertEqual(obj.parent_1, parent_1)
             self.assertEqual(obj.parent_2, parent_2)
 
+    def test_select_related_after_values(self):
+        """
+        Running select_related() after calling values() raises a TypeError
+
+        On #25252 we add a friendly error message about why that happens.
```

Comment:
I don't think adding this line has anything to contribute to the commit. I'd just stick with the first line of the docstring. If you want to include the issue number I usually go with preceding it to the docstring: `#25252 - Running select_related() ...`